mergin/test/test_client.py

```diff
@@ -1952,3 +1952,28 @@ def test_clean_diff_files(mc):
     diff_files = glob.glob("*-diff-*", root_dir=os.path.split(mp.fpath_meta("inserted_1_A.gpkg"))[0])
 
     assert diff_files == []
+
+
+def test_project_metadata(mc):
+    test_project = "test_project_metadata"
+    project = API_USER + "/" + test_project
+    project_dir = os.path.join(TMP_DIR, test_project)
+
+    cleanup(mc, project, [project_dir])
+    # prepare local project
+    shutil.copytree(TEST_DATA_DIR, project_dir)
+
+    # create remote project
+    mc.create_project_and_push(test_project, directory=project_dir)
```

Comment:
I think we do not even need to create such project on the server - we can simply the old json to .mergin/mergin.json and it should be enough to test parsing

Reply:
Fixed.